packages/xo-server/src/api/proxy.js

```diff
@@ -96,15 +96,35 @@ update.params = {
   },
 }
 
-export function deploy({ sr }) {
-  return this.deployProxy(sr._xapiId)
+export function deploy({ network, sr }) {
```

Comment:
What did you change the name?

Reply:
I've wanted to simplify the name but if you see that it's not better i can revert it.

Reply:
It was just a question, you can keep it that way.